source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,233 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data in your MongoDB collections and return computed results.
+
+A powerful way to think of aggregation is like a car factory. Within the car factory is an assembly line, along which
```

Comment:
Editorial nit: I would not claim powerful.  That's in the eye of the beholder, and may turn off some readers who don't find it powerful, or are still confused.  Perhaps just "One way to think of..."

Reply:
Great point, thanks!